lesson08/assignment/video_cli/l8/src/imagepage.vue

```diff
@@ -42,16 +8,16 @@
         <!-- display images in 2 rows -->
         <div class="row">
           <div class="col-sm" v-for="image in images1" :key="image.id">
-            <img :src="image.url" class="img-fluid" alt="Movie themed images">
+            <!-- Updated src to use require() so dynamic links work -->
```

Comment:
Dynamic images in Vue 2 need to be wrapped in a require() method.
https://stackoverflow.com/questions/40491506/vue-js-dynamic-images-not-working